ac1/_sources/sql/filtering.rst

```diff
@@ -7,11 +7,11 @@
 Filtering
 =========
 
-The notion of filtering data should already be familiar to you. Filtering data
-is to look at only a subset of rows, based on some column condition. For
-example, if you have a database containing information for all citizens of the
-USA, a filter could be applied to look only at residents of Texas. You have
-already seen :ref:`how to apply filters in Sheets<filtering_data>`.
```

Comment:
Why did we delete the reference to filters in Sheets? I feel like this is a useful reference.